hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -341,7 +341,11 @@ private void ensurePartitionsLoadedCorrectly(List<String> partitionList) {
         long beginTs = System.currentTimeMillis();
         // Not loaded yet
         try {
-          LOG.info("Building file system view for partitions " + partitionSet);
+          if (partitionSet.size() < 100) {
+            LOG.info("Building file system view for partitions: " + partitionSet);
```

Comment:
Can we just switch to LOG.debug, is the logging really useful?

Reply:
yes, may be we should reconsider the freq of logging here. for eg, log every every 100 partitions or something. not sure we will gain much by logging this for every partition.

Reply:
Converted to a debug log.